stacks/shared-ecs/asg.yml

```diff
@@ -436,6 +436,20 @@ Resources:
       IpProtocol: tcp
       ToPort: 65535
 
+  AsgInstanceKeyPair:
```

Comment:
I'm curious how you get the keypair.  Is it an output in CFN?  Are there security implications to it being visible there forever?

Reply:
It puts the secret in Parameter Store when it's created. it's never available by CFN